modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/useSubtasksActions.tsx

```diff
@@ -31,6 +33,30 @@ const useSubtasksActions = () => {
 	const {modal: completeModal} = useFormModal();
 
 	const actionsRef = useRef([
+		{
+			action: (subtask, mutate) => {
+				testraySubTaskImpl.assignToMe(subtask).then(() => {
+					updateItemFromList(
+						mutate,
+						0,
+						{},
+						{
+							revalidate: true,
+						}
+					);
+				});
+			},
```

Comment:
Suggestion: you can simplify returning directly

```suggestion
		{
			action: (subtask, mutate) => 
				testraySubTaskImpl.assignToMe(subtask).then(() => 
					updateItemFromList(
						mutate,
						0,
						{},
						{
							revalidate: true,
						}
					)
				),
```